ansible/provision-and-execute-tests/playbook-provision-prerelease.yml

```diff
@@ -0,0 +1,58 @@
+- name: (linux) (prerelease) Install CrowdStrike Falcon and NRIA from prod bucket
+  hosts: linux
+  become: true
+  gather_facts: yes
+  vars:
+    # install_crowdstrike_falcon role vars
+    falcon_client_id: "{{ lookup('env', 'CROWDSTRIKE_CLIENT_ID') }}"
+    falcon_client_secret: "{{ lookup('env', 'CROWDSTRIKE_CLIENT_SECRET') }}"
+    falcon_customer_id: "{{ lookup('env', 'CROWDSTRIKE_CUSTOMER_ID') }}"
+    api_base_url: "https://api.laggar.gcw.crowdstrike.com"
+    # newrelic_cli role vars
+    repo_endpoint: "https://nr-downloads-main.s3.amazonaws.com/"
+    recipe: "newrelic-infra"
+    region: "{{ lookup('ansible.builtin.env', 'NEW_RELIC_REGION') }}" # US | EU | STAGING
+    nr_api_key: "{{ lookup('ansible.builtin.env', 'NEW_RELIC_API_KEY') }}"
+    nr_account_id: "{{ lookup('ansible.builtin.env', 'NEW_RELIC_ACCOUNT_ID') }}"
+  roles:
+    - caos.ansible_roles.install_crowdstrike_falcon
+    - caos.ansible_roles.newrelic_cli
```

Comment:
What if we took this whole block out and simply invoked `playbook-provision-staging.yml` (either externally from the controller or just invoke the playbook from here)?